src/org/ods/quickstarter/Pipeline.groovy

```diff
@@ -128,7 +128,7 @@ class Pipeline implements Serializable {
             ]
         }
 
-        def podLabel = "quickstarter-${config.sourceDir}-${config.projectId}-${config.componentId}"
+        def podLabel = "pod-${UUID.randomUUID().toString()}"
```

Comment:
Actually, it would be good to still distinguish it from other pods visually in the UI, so maybe:
```suggestion
        def podLabel = "qs-${UUID.randomUUID().toString()}"
```